src/videotile/DefaultVideoTile.ts

```diff
@@ -58,6 +59,14 @@ export default class DefaultVideoTile implements DevicePixelRatioObserver, Video
 
     if (videoElement.srcObject !== videoStream) {
       videoElement.srcObject = videoStream;
+
+      // In Safari, a hidden video element can show a black screen.
+      // See https://bugs.webkit.org/show_bug.cgi?id=241152 for more information.
+      if (new DefaultBrowserBehavior().requiresVideoPlayWorkaround() && videoElement.paused) {
+        try {
+          videoElement.play();
+        } catch (error) {}
```

Comment:
Should we log the error in case it happens?

Reply:
Yes. But we have to use `console.error()` because this static method in `DefaultVideoTile` does not have access to a logger. What do you think about using `console.error`?

Reply:
I think console.error for now is fine. It is better to log error to screen. We can move console to logger when there are more places we need to log.

Reply:
Actually, `try/catch` block is not needed because we call `HTMLMediaElement.play()` without the `await` keyword.

```js
try {
  videoElement.play();
} catch (error) {
  // Never reached even if play() throws an error
}
```

```js
try {
  await videoElement.play();
} catch (error) {
  // play() throws an error!
}
```

I pushed a revision removing the `try/catch` block.